From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giuliano Procida <gprocida@google.com>
Date: Thu, 9 Jul 2020 17:43:32 +0100
Subject: Enable Clang's -Werror-overloaded-virtual.

Ths warning is no longer triggered and can be reenabled.

	* configure.ac: Remove the special clause that disabled
	-Werror-overloaded-virtual for Clang builds.

Signed-off-by: Giuliano Procida <gprocida@google.com>
Signed-off-by: Matthias Maennich <maennich@google.com>
---
 configure.ac | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 225ac38f245d..904a726ebcc1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -646,12 +646,6 @@ if test x$ENABLE_UBSAN = xyes; then
     CXXFLAGS="$CXXFLAGS -fsanitize=undefined"
 fi
 
-dnl Check if we compile with clang and set flags accordingly
-dnl TODO: fix that warning (spoiler: not trivial)
-if `$CXX -v 2>&1 | grep 'clang version' > /dev/null 2>&1`; then
-    CXXFLAGS="$CXXFLAGS -Wno-error-overloaded-virtual"
-fi
-
 dnl Check if several decls and constant are defined in dependant
 dnl libraries
 HAS_EM_AARCH64=no
